### DISCUSSION TITLE
Update v6.6 - dev.4

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#208] [Pre-Release]
* add `Show un-tagged` tags menu option
* fix Tag Manager rename issue
* result list opened from tags menu will switch to `Search` desktop
* allow ordering of result list popup categories

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.6-dev.4'>Update v6.6 - dev.4</a>.</em>

**Comment by ltguillaume** [comment]
Ordering the long touch category order, works great!

Now that the tags are shown in a menu, I'd like to change the tag names from lower case to starting with a single capital. With dev4, renaming a tag (e.g. `system` to `System`) causes the following:

1. In the _manager_ it loses the icon that was previously chosen for it
2. The tags menu still shows the old lower case tag name
3. Choosing the tag from the tags menu now opens a blank results list

**Reply by TBog** [reply]
1. I'll have to reload the icon then.
2. Because only the tags from the database change. The tags menu still has the old tag. You could uncheck the old tag and check ☑ the new one.
3. as expected

**Reply by ltguillaume** [reply]
>     2. Because only the tags from the database change. The tags menu still has the old tag. You could uncheck the old tag and check ☑ the new one.

That's pretty confusing: it acts like renaming an existing tag is like I'm creating a new tag and moving the apps into the new tag, while keeping the old tag. That's not what a rename operation should look like for the end user.

**Reply by TBog** [reply]
I understand. I'll replace the tag from the tags menu as well.
Thanks for noticing and reporting the issue.

**Reply by TBog** [reply]
Should be fixed in the latest release, can you please verify?

**Reply by ltguillaume** [reply]
> Should be fixed in the latest release, can you please verify?

I ran into some problems, a crash, possibly caused by remnants of the previous version in the database. See discussion of dev5.

**Comment by ltguillaume** [comment]
![image](https://user-images.githubusercontent.com/4777345/135508700-e53c2aed-426e-4e8d-a982-2eaf6ca98c60.png)

Perhaps you could apply the same kind of spacing in the tags menu as used in the long-press menu? Currently:

1. It's difficult to press the correct item
2. If you choose to show the tag icons, the item "Un-tagged" is really small in comparison.

**Reply by TBog** [reply]
Should be fixed in the latest release, can you please verify?
